examples/lines_bars_and_markers/stairs_demo.py

```diff
@@ -4,26 +4,23 @@
 ===========
 
 This example demonstrates the use of `~.matplotlib.pyplot.stairs`
-for histogram and histogram-like data visualization and an associated
-underlying `.StepPatch` artist, which is a specialized version of
-`.PathPatch` specified by its bins and edges.
+for piecewise constant functions. A common use case is histogram and
```

Comment:
```suggestion
for stepwise constant functions. A common use case is histogram and
```
You used "stepwise" below, which is less precise, but I think more colloquial.